src/main/java/com/health/healther/member/domain/model/Member.java

```diff
@@ -0,0 +1,79 @@
+package com.health.healther.member.domain.model;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import org.hibernate.envers.AuditOverride;
+
+import com.health.healther.member.domain.type.LoginType;
+import com.health.healther.member.domain.type.MemberStatus;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor
+@AuditOverride(forClass = BaseEntity.class)
+public class Member extends BaseEntity {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	private String name;
+
+	private String nickName;
+
+	private String phone;
+
+	@Enumerated(EnumType.STRING)
+	private MemberStatus memberStatus;
+
+	private String oauthId;
+
+	@Enumerated(EnumType.STRING)
+	private LoginType loginType;
```

Comment:
```suggestion
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@Column(name = "MEMBER_ID")
	private Long id;

	@Column(name = "MEMBER_ID")
	private String name;

	@Column(name = "NICKNAME")
	private String nickName;

	@Column(name = "PHONE")
	private String phone;

	@Enumerated(EnumType.STRING)
	@Column(name = "MEMBER_STATUS")
	private MemberStatus memberStatus;

       	@Column(name = "OAUTH_ID")
	private String oauthId;

	@Enumerated(EnumType.STRING)
	@Column(name = "LOGIN_TYPE")
	private LoginType loginType;
```
위와 같이 Column 어노테이션을 사용하여 네임을 직접 지정해주는거로 통일하면 좋을 것 같아요!

Reply:
Suggested change 어떻게 하셨는지가 오히려 궁금하네요
깃허브 관련 공부도 할게 많네요 🙄
좋은 의견이라 생각합니다! 반영할게욥